docs/README.md

```diff
@@ -12,17 +45,29 @@ There are several ways to get started using .NET Interactive with Jupyter, inclu
 * [Create and run .NET notebooks on your machine](NotebookswithJupyter.md).
 * [Share .NET notebooks online using Binder](CreateBinder.md).
 
-### Visual Studio Code
 
-Work is underway to add support for the new Visual Studio Code [native notebook feature](https://code.visualstudio.com/updates/v1_45#_github-issue-notebook). While we are still in the early stages of this effort, if you'd like to experiment with it you can find instructions [here](../src/dotnet-interactive-vscode/README.md). 
+## REPLs
+
+.NET Interactive can be used as the execution engine for REPLs. For an example using a CLI, see [.NET REPL](https://github.com/jonsequitur/dotnet-repl). In addition, .NET REPL can actually be used to set up automation for your Polyglot Notebooks. 
+ 
+
+## Acknowledgements 
+
+The multi-language experience of .NET Interactive is truly a collaborative efforts amongst other groups at Microsoft. We'd like to thank the following teams for contributing their time and expertise to helping light up functionality for other languages. 
+
+- **PowerShell Team:** PowerShell
+- **Azure Data/SQL Team:** SQL, KQL
+
+
+## Other
 
 ### Small factor devices
 
 We support running on devices like Raspberry Pi and [pi-top [4]](https://github.com/pi-top/pi-top-4-.NET-Core-API). You can find instructions [here](small-factor-devices.md).
 
 ### Telemetry
 
-Telemetry is collected when .NET Interactive is started. Once .NET Interactive is running, we do not collect telemetry from any further user actions. The telemetry is anonymous and reports only the values for a specific subset of the verbs in the .NET Interactive CLI. Those verbs are:
+Telemetry is collected when .NET Interactive is started. Once .NET Interactive is running, we collect packages imported into the notebook and the languages used to run individual cells. We do not collect any additional code from cells. The telemetry is anonymous and reports only the values for a specific subset of the verbs in the .NET Interactive CLI. Those verbs are:
```

Comment:
```suggestion
Telemetry is collected when .NET Interactive is started. Once .NET Interactive is running, we collect the names of packages imported into the notebook and the languages used to run individual cells. We do not collect any additional code from cells. The telemetry is anonymous and reports only the values for a specific subset of the verbs in the .NET Interactive CLI. Those verbs are:
```

Reply:
We collect hash from the language and package name, not the clear value. Should not this be in the readme?

Reply:
Not a bad call. Basically, we collect these values but we don't know what they are unless they match specific well-known hashes, i.e. only the packages we know to look for are identifiable.